src/sidebar/components/test/share-annotations-panel-test.js

```diff
@@ -132,30 +141,49 @@ describe('ShareAnnotationsPanel', () => {
         testCase.visibilityPattern
       );
     });
+
+    context('document URI cannot be shared', () => {
+      it('renders explanatory text about inability to share', () => {
+        fakeIsShareableURI.returns(false);
+
+        const wrapper = createShareAnnotationsPanel();
+
+        const panelEl = wrapper.find('.share-annotations-panel');
+        assert.include(panelEl.text(), 'These annotations cannot be shared');
+      });
+    });
   });
 
   describe('web share link', () => {
     it('displays web share link in readonly form input', () => {
       const wrapper = createShareAnnotationsPanel();
 
       const inputEl = wrapper.find('input');
-      assert.equal(
-        inputEl.prop('value'),
-        'https://hyp.is/go?url=https%3A%2F%2Fwww.example.com&group=testprivate'
-      );
+      // assert.equal(
+      //   inputEl.prop('value'),
+      //   'https://hyp.is/go?url=https%3A%2F%2Fwww.example.com&group=testprivate'
+      // );
```

Comment:
Thanks for your patience on this. I've added it to my list of "stuff I sometimes overlook in PRs" to see if I can be more careful about this in the future. There's no straightforward way to lint for it, so it might just rely on me being a little more attentive!